### PR TITLE
Prompting the user for uniqname and password

### PR DESCRIPTION
**Pull request by NickLamping**
I changed the script so that it prompts the user for his or her uniqname and password. It might be a bit tedious when running the script over and over again. It's a small change but it fixes some security issues with having your password in plaintext on your machine.

### CODE REVIEW

File: leccap_dl.py [hunk 1]
```diff
@@ -26,6 +24,8 @@ def parse_args():
 def main():
     args = parse_args()
 
+    UNIQNAME = raw_input("Uniqname: ")
+    PASSWORD = getpass.getpass("Password: ")
```
Comment (najork) [inline leccap_dl.py]: Add newline before comment

File: leccap_dl.py [hunk 2]
```diff
@@ -26,6 +24,8 @@ def parse_args():
 def main():
     args = parse_args()
 
+    UNIQNAME = raw_input("Uniqname: ")
```
Comment (najork) [inline leccap_dl.py]: Since these are no longer global fields, make lowercase

**Comment by najork** [comment]
Other than the two minor comments, this looks good! I'll merge after you push the changes.

**Comment by najork** [comment]
Good point regarding having the plaintext password saved on your machine, but I want to point out that the larger security issue is that you might accidentally push your password to a public repo

**Comment by NickLamping** [comment]
True, that would be very bad. I believe this last commit addresses your comments.

**Pull request merged**